components/shopify_partner/shopify_partner.app.js

```diff
@@ -0,0 +1,83 @@
+require("graphql/language");
+const get = require("lodash/get");
```

Comment:
Now that sources and workflows support Node 14, we prefer using the nullish coalescing operator in place of `lodash/get` to remove the dependency. I looked at the examples where you use `get` and I think nullish coalescing will work for each, but lmk if not.

Reply:
Yup agreed, just was dealing with that `undefined` bug and saw lodash.get was still being referencing in docs/other actions. Wasn't sure if that's the direct we were heading.

Reply:
Actually, it kind of makes sense to use `get` in this case because lodash's get offers nested pathing, which I don't believe is available out of the box with ES6.

For example, the cursor for pagination for the uninstalls source is:

`const cursorPath = 'app.events.edges[0].cursor'`

It's not possible - afaik - to path in ES6 with this:

```javascript
const cursorPath = 'app.events.edges[0].cursor'
const cursor = response[cursorPath];
```

So in this case, get is nice because it supports nested array & object traversal with that dot notation.